x-pack/plugins/ml/common/util/anomaly_utils.js

```diff
@@ -27,15 +28,25 @@ const DISPLAY_TYPICAL_FUNCTIONS = ['count', 'distinct_count', 'lat_long', 'mean'
 // for the supplied normalized anomaly score (a value between 0 and 100).
 export function getSeverity(normalizedScore) {
   if (normalizedScore >= 75) {
-    return 'critical';
+    return { id: 'critical', label: i18n.translate('xpack.ml.anomaliesTable.severity.criticalLabel', {
```

Comment:
All these IDs should start with `xpack.ml.anomalyUtils` rather than `xpack.ml.anomaliesTable` as they are used in more places than just the Anomalies table component.